internal/kernel/t_api/status.go

```diff
@@ -23,6 +23,7 @@ const (
 	StatusTaskAlreadyCompleted   StatusCode = 40306
 	StatusTaskInvalidCounter     StatusCode = 40307
 	StatusTaskInvalidState       StatusCode = 40308
+	StatusFencingTokenInvalid    StatusCode = 40309
```

Comment:
Can we use code 41200, which would map to a [412 precondition failed](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status/412)